cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
@kkrentz I've just tried your `potr` tests. I can't get a single packet through. I only get NAKs on the sender, and the receiver gets nothing:
```
Contiki-3.x-2589-g125569b
Zolertia Firefly platform
Rime configured with address 00:12:4b:00:06:16:0f:9c
 Net: nullnet
 MAC: Sec-MAC
 RDC: nordc
OK 0 / COLLISION 0 / NOACK 1 / DEFERRED 0 / ERR 0 / ERR_FATAL 0
OK 0 / COLLISION 0 / NOACK 2 / DEFERRED 0 / ERR 0 / ERR_FATAL 0
OK 0 / COLLISION 0 / NOACK 3 / DEFERRED 0 / ERR 0 / ERR_FATAL 0
```

I've changed the addresses in both the sender and the receiver to the Rime addresses. I've also defined `RTIMER_CONF_GUARD_TIME` to `7` in `contiki-conf.h` as you did it for `openmote-cc2538`. Is there anything else to do?